### PR TITLE
Add draft release notes for first RC of 1.5.0.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: _releases/1.5.0.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+ * Add support for querying multiple LDAP servers ([GUACAMOLE-957](https://issues.apache.org/jira/browse/GUACAMOLE-957))
+ * {:.bug} SSL communication by mariadb connector/J is not possible ([GUACAMOLE-1453](https://issues.apache.org/jira/browse/GUACAMOLE-1453))
+
+### Protocol support / guacd
+
+ * Add support for OpenSSH private key format ([GUACAMOLE-745](https://issues.apache.org/jira/browse/GUACAMOLE-745))
+ * Add support for ED25519 SSH keys ([GUACAMOLE-746](https://issues.apache.org/jira/browse/GUACAMOLE-746))
+ * {:.bug} RDP "reconnect" resizing breaks RDPDR ([GUACAMOLE-876](https://issues.apache.org/jira/browse/GUACAMOLE-876))
+ * {:.bug} RDP print process may hang ([GUACAMOLE-1115](https://issues.apache.org/jira/browse/GUACAMOLE-1115))
+ * {:.bug} An improper locking bug(e.g., resource leak) due to unreleased lock ([GUACAMOLE-1416](https://issues.apache.org/jira/browse/GUACAMOLE-1416))
```
Comment (necouchman) [inline _releases/1.5.0.md:185]: #nitpick - Should probably be a space between "bug" and the opening parenthesis.

**Review by necouchman — CHANGES_REQUESTED** [review]
I know it's still in draft mode, but looks pretty good. One minor nitpick - missing space - noted below.

File: _releases/1.5.0.md (hunk 1 above)
Comment (mike-jumper) [reply _releases/1.5.0.md:185]: True - I'll fix that. These were just automatically scraped from JIRA. ;)

**Comment by mike-jumper** [comment]
Un-drafted!

**Comment by jmuehlner** [comment]
@necouchman Github still shows that you have requested changes - can you confirm that your concerns have been addressed?

**Review by necouchman — APPROVED** [review]
All good, here.

**Pull request merged**